packages/flutter/lib/src/material/search.dart

```diff
@@ -188,6 +188,12 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.leading], the intended use for the return value of this method.
   Widget? buildLeading(BuildContext context);
 
+  /// [AppBar.automaticallyImplyLeading]
```

Comment:
Here we should use `/// {@macro flutter.material.appbar.automaticallyImplyLeading}` . 

And in `app_bar.dart`:
https://github.com/flutter/flutter/blob/7f20e5d18ce4cb80c621533090a7c5113f5bdc52/packages/flutter/lib/src/material/app_bar.dart#L277-L284

change:
`leading` -> `AppBar.leading`, and `title` -> `AppBar.title`.

Reply:
updated